{
  "tweet": {
    "edit_info": {
      "initial": {
        "editTweetIds": [
          "1733603868666413494"
        ],
        "editableUntil": "2023-12-09T22:45:45.000Z",
        "editsRemaining": "5",
        "isEditEligible": false
      }
    },
    "retweeted": false,
    "source": "<a href=\"https://mobile.twitter.com\" rel=\"nofollow\">Twitter Web App</a>",
    "entities": {
      "hashtags": [],
      "symbols": [],
      "user_mentions": [],
      "urls": []
    },
    "display_text_range": [
      "0",
      "139"
    ],
    "favorite_count": "0",
    "in_reply_to_status_id_str": "1733601783417909622",
    "id_str": "1733603868666413494",
    "in_reply_to_user_id": "601449312",
    "truncated": false,
    "retweet_count": "0",
    "id": "1733603868666413494",
    "in_reply_to_status_id": "1733601783417909622",
    "created_at": "Sat Dec 09 21:45:45 +0000 2023",
    "favorited": false,
    "full_text": "ok I don't know why but if I have both process.on('close') and process.on('exit'), the process terminates immediately and without error lol",
    "lang": "en",
    "in_reply_to_screen_name": "zemnmez",
    "in_reply_to_user_id_str": "601449312"
  }
}